lambdatimes/src/components/Content/Content.js

```diff
@@ -48,8 +55,12 @@ export default class Content extends Component {
           `selectedTab` that includes the currently selected tab
           and `selectTabHandler` that includes the function to change the selected tab
         */}
-        <Tabs tabs={this.state.tabs} />
-        <Cards cards={this.filterCards()} />
+        <Tabs
+          tabs={this.state.tabs}
+          selectedTab={this.state.selected}
+          selectTabHandler={this.changeSelected}
+        />
+        <Cards cards={ this.state.selected !== 'all' ? this.state.cards.filter(card => card.tab === this.state.selected) : this.state.cards } />
```

Comment:
All of this logic should returned be in the filterCards method and line 63 should be:
`<Cards cards={this.filterCards()} />`
Great job with the logic and ternary statements tho